src/walletSdk/Watcher/index.ts

```diff
@@ -112,79 +112,78 @@ export class Watcher {
       this._watchAllTransactionsRegistry[assetCode] = true;
     }
 
-    this.anchor.getTransactionsForAsset({ 
-      authToken, 
-      assetCode, 
-      lang, 
-      kind, 
-      noOlderThan 
-    }).then((transactions: AnchorTransaction[]) => {
+    this.anchor
```

Comment:
Instead of Anchor you can pass Sep24 class in the constructor now

Reply:
thoughts on keeping it as anchor? Since the watcher needs to access the language field (and wouldn't be able to sep24.anchor.language since anchor is private)

Reply:
Good point